.github/CONTRIBUTING.md

```diff
@@ -245,11 +245,10 @@ Bad: Can't parse input x
 Good: Cannot parse input x
 ```
 
-7. Messages should use a colon when providing additional information:
+7. Messages should not use a period when providing additional information:
```

Comment:
I feel this obscures that we prefer colon. Can we mention both colon and period?

Reply:
Updated it to read:

```
7. Messages should use a colon when providing additional information. Periods
   should never be used. Other punctuation may be used as needed:
```